utils/python/CIME/case_st_archive.py

```diff
@@ -285,7 +285,12 @@ def _archive_restarts(case, archive, archive_entry,
                             logger.debug("copying %s to %s" %(srcfile, destfile))
                     else:
                         srcfile = os.path.join(rundir, restfile)
-                        logger.info("removing interim restart file %s" %srcfile)
+                        if (os.path.isfile(srcfile)):
+                            try:
+                                os.remove(srcfile)
+                                logger.info("removing interim restart file %s" %srcfile)
+                            except OSError as e:
+                                logger.warn("interim restart file %s does not exist " %srcfile)
```

Comment:
Shouldn't this warning be that the file could not be removed?  It won't even be in this code block if the file didn;t exist (per the if statement at 288)